onmt/utils/loss.py

```diff
@@ -261,6 +261,51 @@ def _bottle(self, _v):
     def _unbottle(self, _v, batch_size):
         return _v.view(-1, batch_size, _v.size(1))
 
+    def ignore_prompt(
+        self, batch, response_token="▁Response", response_right_pattern_length=3
```

Comment:
Do we want a more specific placeholder, e.g. `｟_response_｠`. This "▁Response" feels too generic and we might have some cases where we'd unwillingly mask some content that should not be masked.

Reply:
Or at least make it configurable via an opt flag.